src/main/views/courts/tabs/postcodesContent.njk

```diff
@@ -0,0 +1,138 @@
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+
+<h2 class="govuk-heading-l">Postcodes</h2>
+
+{{ govukDetails({
+  summaryText: "Help with postcodes",
+  html: "Only full or partials postcodes are accepted. For example, SA1 1AA, or SA1."})
+}}
+
+{% if updated === true and not errors|length %}
+  {{ govukPanel({
+    titleText: "Postcodes updated"
+  }) }}
+{% endif %}
+
+{% if errors|length %}
+  {{ govukErrorSummary({
+    titleText: "There is a problem",
+    errorList: errors
+  }) }}
+{% endif %}
+
+<input type="hidden" name="postcodesEnabled" id='postcodesEnabled' value="{{ isEnabled }}">
+
+{% if (areasOfLaw|length) %}
+
+  {% if searchValue == '' and updated and errors|length %}
+    {% set addError = { text: 'One or more postcodes are required.' } %}
+  {% elif searchValue != '' and updated and errors|length %}
+    {% set addError = { text: 'Please make sure that postcodes are not already added for this court.' } %}
+  {% endif %}
+
+  {{ govukInput({
+    label: {
+      text: "Add postcodes:",
+      classes: "govuk-label--m",
+      isPageHeading: false
+    },
+    hint: {
+      text: "Please add a comma separated list of postcodes.
+      These can be case insensitive, and spaces are not significant."
+    },
+    value: searchValue if updated,
+    id: "addNewPostcodes",
+    name: "addNewPostcodes",
+    errorMessage: addError
+  }) }}
+
+  <input id="existingPostcodesInput" name="existingPostcodesInput" type="hidden" value= {{ postcodes }}>
+  <input id="courtTypesInput" name="courtTypesInput" type="hidden" value= {{ courtTypes }}>
+  <input id="areasOfLawInput" name="areasOfLawInput" type="hidden" value= {{ areasOfLaw }}>
+
+  {{ govukButton({
+    text: "Add",
+    name: "addPostcodes",
+    type: "button",
+    classes: "addPostcodes"
+  }) }}
+
+  {% if postcodes|length %}
+
+    {% set postcodesCheckboxItems = [] %}
+    {% for postcode in postcodes %}
+      {% set postcodesCheckboxItems =
+        (postcodesCheckboxItems.push({'text': postcode, 'value': postcode, 'id': postcode}), postcodesCheckboxItems) %}
+    {% endfor %}
+
+    <h2 class="govuk-heading-m">Delete or move postcodes:</h2>
+
+    {{ govukCheckboxes({
+      idPrefix: "postcodes-select-all",
+      name: "postcodesSelectAllItems",
+      classes: "govuk-checkboxes--medium",
+      items: [
+        {
+          value: "selectAll",
+          text: "Select all"
+        }]
+    }) }}
+
+    <div id="postcodesList" class="govuk-grid-row visible-scrollbar scrollable", role="scrollbar">
+      {%- for items in postcodesCheckboxItems | slice(2) %}
+        <div class="govuk-grid-column-one-half">
+            {{ govukCheckboxes({
+              idPrefix: "postcodes-checkbox",
+              name: "postcodesCheckboxItems",
+              classes: "govuk-checkboxes--medium",
+              items: items
```

Comment:
Is the loop required? I thought the items property was an array? https://design-system.service.gov.uk/components/checkboxes/#options-checkboxes-example--items

Reply:
Yeah its Majid's fancy way of having the list appear separately in different rows on the select box. Otherwise its all in one massive column. Refer to the slice(2) bit :-)